pkg/registry/servicecatalog/broker/strategy_test.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package broker
+
+import (
+	"testing"
+
+	sc "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
+)
+
+// TestBrokerStrategyTrivial is the testing of the trivial hardcoded
+// boolean flags.
+func TestBrokerStrategyTrivial(t *testing.T) {
+	if createStrategy.NamespaceScoped() {
```

Comment:
In the case of `NamespaceScoped`, you definitely want create and update to perform the same for things to work right because it affects how the etcd path is calculated.

> The shortest question is, why is "NamespaceScoped" defined where it is, and not elsewhere? I fear the answer is deep inside of apimachinery.

I think the encompassing question is 'why are there several small interfaces instead of one that handles create, update, and delete', right?